src/components/20-molecules/datepicker/index.js

```diff
@@ -1,28 +1,27 @@
 /* eslint-disable import/no-extraneous-dependencies */
+import AXADropdown from '@axa-ch/dropdown';
```

Comment:
Something changed at that file? many changes, but just reordering?

Reply:
No, there aren't any changes here, except the reordered imports (that was my commit to re-run the checks).